### ISSUE TITLE
unable to tell voting status with a screen reader

### ISSUE THREAD

**Issue by fastfinge** [LemmyNet/lemmy-ui#1068]
### Issue Summary

As a screen reader user, I am unable to tell if I have successfully upvoted or downvoted content or comments, because nothing is announced when the button is pressed, and the status of the button does not change in any way.

### Steps to Reproduce

1. Log into Lemmy
2. Run A screen reader (NVDA from nvda-project.org or Narrator on windows, or Voiceover on OS X).
3. upvote or downvote any content or comment
4. Notice that the only way you know your vote happened is by looking to see if the upvote number changed.

### Technical details

A good fix for this might be making the upvote and downvote buttons toggles. Once upvote or downvote is pressed, the button could toggle to "pressed". That way a screen reader would announce something like "toggle button pressed upvote" or "toggle button pressed downvote" for content the user has voted up or down on, or "toggle button not pressed" for both buttons if the user hasn't voted.

**Comment by fastfinge** [comment]
Wow, another quick fix; Thanks!
